mqt/mqt.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from __future__ import (
+    absolute_import, division, print_function, with_statement,
+    unicode_literals
+)
+
+
+class Mqt(object):
```

Comment:
is this the placeholder addressing "NOTHING can be moved in the meanwhile we must maintain the scripts as they are AND create new files to make them Importable Classes to be used in the click CLI interface."?

Meaning that will have a `Linter` class for instance?

Reply:
cookiecutter part.

Reply:
@simahawk sorry for the last unexplained answer, Yes, that's what I am trying I will use that class for the creation of the environmental decisions in order to have such place already set to avoid design mistakes in the move of the other scripts.

Reply:
>> Meaning that will have a Linter class for instance? You can see on the current status what that class will be used for, that make sense for you?